docs/projects/fractions/_posts/2019-10-10-c-plus-plus.md

```diff
@@ -0,0 +1,251 @@
+---
+title: fractions in C++
+layout: default
+last-modified: 2019-10-10
+featured-image:
+tags: [c-plus-plus, fractions]
+authors:
+  - niraj-kamdar
```

Comment:
Make sure you add yourself to the author file. https://github.com/TheRenegadeCoder/sample-programs/blob/master/docs/_data/authors.yml

Reply:
I'm going to mention this again.

Reply:
If you add yourself to the authors file, we'll be good to go.